packages/core/src/configuration.ts

```diff
@@ -745,6 +745,9 @@ export const transitions = {
 	pie_slice_hover: {
 		duration: 100
 	},
+	pie_chart_titles: {
```

Comment:
aren't they just callouts?

Reply:
the same transition needs to be on the callouts as the chart titles. I'm grouping together callouts and titles (since callout cannot exist without a corresponding title). this gets applied to the title texts and the lines

Reply:
ok great